### PR TITLE
Add icons for Views and Replies, and Last Activity date

### PR DESCRIPTION
**Pull request by communiteq**
As discussed in https://meta.discourse.org/t/add-icons-for-views-and-replies-and-last-activity-date-in-topic-cards/312505 this PR adds icons for views, replies and last activity.

### CODE REVIEW

File: common/common.scss [hunk 1]
```diff
@@ -173,20 +188,14 @@
...
-        align-items: center;
-        gap: 0.25em;
-        &-number {
-          color: var(--love);
-          font-weight: bold;
-          font-size: var(--font-down-1-rem);
-        }
-
-        .d-icon {
-          color: var(--love);
```
Comment (pmusaraj) [inline common/common.scss:187]: We've regressed on the color for the heart icon, can we restore that? Or does it look poorly if its color is different from the other icons?

**Review by pmusaraj — COMMENTED** [review]
Looks good overall, can you run the linter as well?

File: common/common.scss (hunk 1 above)
Comment (communiteq) [reply common/common.scss:187]: It was on purpose, it indeed looks really stupid if it's the only colored icon. Sorry for not mentioning that explicitly. 
I wasn't sure how to deal with this, remove it by default seemed the best idea.

Comment (quimgil) [reply common/common.scss:187]: Hi, for what is worth I agree with the removal of the red color. Initially Topic Cards had only one icon, and having it red was ok. But in Topic List Thumbnails none of the icons deviates from gray, which makes sense, and this PR follows that consistency. 

For what is worth, this is how the four icons looks like, and it looks very good imho:

image

**Comment by communiteq** [comment]
> Looks good overall, can you run the linter as well?

I'm not sure why the linter failed in the Github action, it works for me locally?

File: common/common.scss (hunk 1 above)
Comment (pmusaraj) [reply common/common.scss:183]: Sorry about the delay here, folks, I was out on vacation. 

I found a way to add the like icon color but only when it is displayed on its own. Goal here was to keep existing behaviour intact.

**Review by pmusaraj — APPROVED** [review]

**Comment by pmusaraj** [comment]
Thanks again @communiteq !

File: common/common.scss (hunk 1 above)
Comment (communiteq) [reply common/common.scss:183]: That's a brilliant solution!! 👍

**Comment by communiteq** [comment]
And thank you for merging!

**Pull request merged**